internal/agent/device/fileio/fileio_unix.go

```diff
@@ -0,0 +1,110 @@
+//go:build !windows
+
+package fileio
```

Comment:
Is all the windows/file compatibility here necessary after the change in cli/certificate?

Reply:
fileio is internal to the agent, was something in the cli depending on it?

Reply:
yes, removed it